.circleci/check_golint.sh

```diff
@@ -4,7 +4,7 @@
 # golint print issues it found in source files
 # count lines with wc and check that N lines was in output
 result=$($GOPATH/bin/golint ./... | wc -l)
-if [[ $result -gt 10 ]]; then
+if [[ $result -gt 12 ]]; then
```

Comment:
I surprised, that now tests didn't pass due to golint check with 20+ checks on master branch. As I think, maybe he updated some checks after new versions of golang... Previously we left 10 entries from golint which suggest to remove redundant words from names like `type AcraCensorInterface interface` (it suggested remove Interface suffix), or remove `JSON` from `ZoneInfoJSON` because it declared in `json` package ). Now such suggestions 12 items...